src/SFA.DAS.EmployerCommitmentsV2.Web/Validators/SelectProviderViewModelValidator.cs

```diff
@@ -0,0 +1,27 @@
+﻿using FluentValidation;
+using SFA.DAS.EmployerCommitmentsV2.Web.Models.CreateCohort;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Validators
+{
+    public class SelectProviderViewModelValidator : AbstractValidator<SelectProviderViewModel>
+    {
+        public SelectProviderViewModelValidator()
+        {
+            RuleFor(x => x.ProviderId)
+                .Must(BeALongGreaterThanZero)
+                .WithMessage("Check UK Provider Reference Number");
```

Comment:
you don't have any tests to verify this message.

Reply:
added unit test to verify error